auth0-lock.ts

```diff
@@ -9,13 +10,20 @@ declare var Auth0Lock: any;
 export class AuthService {
   // Configure Auth0
   lock = new Auth0Lock(this.clientId, this.domain, {
-    closable: false // Users are required to log in
+    closable: false, // Users are required to log in
+    auth: {
+      redirectUrl: (this.redirectPath == null ? location.origin : location.origin + this.redirectPath),

```

Comment:
That would probably work as well, string concatenation in Typescript should do the same thing either way.